.eslintrc.cli.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://json.schemastore.org/eslintrc",
+  "extends": [".eslintrc.json"],
+  "overrides": [
+    {
+      // This override adds type-checked rules.
+      "files": ["**/*.ts", "**/*.tsx"],
+      // Linting with type-checked rules is very slow and needs a lot of memory,
+      // so we exclude non-essential files.
+      "excludedFiles": ["examples/**/*", "test/**/*", "**/*.d.ts"],
+      "parserOptions": {
+        "project": true
+      },
+      "rules": {
+        // TODO: enable in follow-up PR
+        "@typescript-eslint/no-floating-promises": "off"
```

Comment:
Are you planning on fixing every violation in a follow-up or just flag them via eslint-disable directive first? If you want to go the eslint-disable route, you can use `npx eslint-focus @typescript-eslint/no-floating-promises ./some-folder --allowInlineConfig --fix --fix-type add-disable-directive` from https://github.com/eps1lon/eslint-focus

That would also give you a good hint about performance of this single rule since `eslint-focus` runs with only the specified rules enabled.

Reply:
Not sure yet, need to look at the findings first. Draft PR is here: #67690